models/biocro/R/met2model.BIOCRO.R

```diff
@@ -16,23 +16,60 @@
 ##' @title Write BioCro met files
 ##' @param in.path path on disk where CF file lives
 ##' @param in.prefix prefix for each file
-##' @param outfolder location where model specific output is written.
-##' @param ... can pass lat, lon, start.date and end.date
-##' @return OK if everything was succesful.
+##' @param outfolder location where model specific output is written
+##' @param lat, lon Site latitude and longitude
+##' @param start_date, end_date Date range to convert. Each year will be written to a separate file
+##' @param ... other arguments passed from PEcAn, currently ignored
+##' @return invisibly, a dataframe of information about the written file.
 ##' @export
 ##' @author Rob Kooper, David LeBauer
 ##-------------------------------------------------------------------------------------------------#
-met2model.BIOCRO <- function(in.path, in.prefix, outfolder, overwrite = FALSE, ...) {
-  ncfiles <- dir(in.path, full.names = TRUE, pattern = paste0(in.prefix, "*.nc$"), 
-                 all.files = FALSE, recursive = FALSE)
-  metlist <- list()
-  for (file in ncfiles) {
-    met.nc <- ncdf4::nc_open(file)
-    tmp.met <- load.cfmet(met.nc, lat = lat, lon = lon, start.date = start.date, 
-                          end.date = end.date)
-    metlist[[file]] <- cf2biocro(tmp.met)
+met2model.BIOCRO <- function(in.path, in.prefix, outfolder, overwrite = FALSE,
+                             lat, lon, start_date, end_date, ...) {
+  library(PEcAn.all)
+  dir.create(file.path(outfolder), recursive = TRUE, showWarnings = FALSE)
+  years_wanted <- lubridate::year(start_date):lubridate::year(end_date)
+  out.prefix <- "BIOCRO_MET_INPUT"
```

Comment:
@ankurdesai is there an output naming convention to follow here? Currently produces files named like `/home/carya/output/dbfiles/AmerifluxLBL_BIOCRO_site_1-35//BIOCRO_MET_INPUT.2007.csv`